packages/grid/src/types/index.ts

```diff
@@ -219,3 +221,14 @@ export interface ISplitterProps extends HTMLAttributes<HTMLDivElement> {
   /** Determines splitter orientation within a pane */
   orientation?: Orientation;
 }
+
+export interface ISplitterButtonProps
+  extends Omit<
+    IIconButtonProps,
+    'isRotated' | 'isPrimary' | 'isDanger' | 'isBasic' | 'focusInset' | 'isNeutral' | 'size'
```

Comment:
Are we positive that `focusInset` should be knocked-out here? There may be tighter visual layouts where this focus treatment is desirable.

Reply:
@lucijanblagonic told me no focus inset

Reply:
Since button is only `24px`, has shadow and floating above the border and other UI, I thought that `focusInset` was not needed. Tables have overflow buttons with `focusInset` but are also `28px`. I don't have a strong opinion on this, but that was my initial assumption.